release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,62 @@
 
 This section highlights new features in {product} {product-version}.
 
-== Support for Role-Based Access Control 
+== Change in productized version  
 
-The Role-Based Access Control (RBAC) feature in {product} introduces an administrator role and optimizes access control through organizational structures such as teams, groups, and users.
+{product} is now a fully supported, enterprise-grade productized version of upstream Backstage v1.23.4. 
 
-The RBAC features REST API endpoints that you can use to manage the permissions and roles in {product-short}.
 
-Using the RBAC REST API, you can perform the following actions:
+== Ability to manage role-based access controls (RBAC) using the web interface  
 
-* Retrieve information about all permission policies or specific permission policies, or roles.
-* Create, update, or delete a permission policy or a role.
-* Retrieve permission policy information about static plugins.
+As an administrator, you can now use {product-short} to assign specific roles and permissions to individual users or groups. Using the {product-short} web interface, you can perform the following actions:
 
-[NOTE]
-====
-RBAC is only supported on core {product} features including catalog, software templates, and RBAC plugin.
-====
+* Creating a role 
+* Editing a role and related permissions
+* Deleting a role
 
-== Support for enabling plugins dynamically
 
-The dynamic plugins feature enables you to sideload dynamic plugins into your {product-short} instance without having to recompile your code or rebuild the container.
+==  Migration of {product} to the new backend system  
 
-{product-short} is pre-loaded with a selection of dynamic plugins. Most of these dynamic plugins are disabled by default due to the need for mandatory configuration. For a complete list of dynamic plugins that are included in this release of {product}, see the dynamic plugins matrix in the link:{LinkAdminGuide}#dynamic-plugins-matrix[Dynamic Plugins Matrix] section in the _Administration guide for {product}_.
+{product} {product-version} is now migrated to the Backstage new backend system. This migration in combination with the dynamic plugins feature allows you to extend your backend plugins with an unprecedented ease. With this migration, you might notice the following functionality-related changes in the {product-short} application:
```

Comment:
Is it correct to refer to "the **Backstage** new backend system"? 

@davidfestal can you confirm please?

Reply:
Yes

Reply:
I don't think that mentioning the underlying upstream project is a problem. If that what you are asking. 

Maybe it is just me, but the "_This migration in combination with the dynamic plugins feature allows you to extend your backend plugins with an unprecedented ease_" sentence sounds weird to me.
I've noticed this style in other places in release notes as well. It is sounds like marketing BS mixed with sales pitch :-/ It is something that I wouldn't expect in release notes. But it could be just me. I'm really alergic to empty marketing terms like "an unprecedented ease"

Reply:
Yes, I'd agree that the customer may have a different view of the "unprecedented ease" statement. It's probably better to remove this phrase or the sentence altogether.

Reply:
Removed this sentence